src/print_graph.cc

```diff
@@ -11,6 +11,208 @@
 namespace celerity {
 namespace detail {
 
+	namespace {
```

Comment:
Why the anonymous namespace? The names are already hidden from the public in `celerity::detail`. If you feel like this should be scoped further, I would go for something explicit like `celerity::detail::print_graph_detail`.

Also let's try to avoid nesting namespace declarations because it will cause unnecessary indentation with our current clang-format config.